src/App.js

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
-import Signup from './components/signup/SignUp';
-import Signin from './components/signin/SignIn';
+import SignUp from './components/signup/SignUp';
+import SignIn from './components/signin/SignIn';
+import Home from './components/home/Home';
 
 
 function App() {
   return (
     <Router>
         <Switch>
             <Route exact path="/"
-            component={Signin} />
-            <Route  path="/signup"
-            component={Signup} />
+            component={SignIn} />
+            <Route  path="/SignUp"
```

Comment:
paths should be in lowercase. eg `/signup`

Reply:
Done